mango_base/mango/dashboards/time_series_utils/data_processing.py

```diff
@@ -0,0 +1,38 @@
+import pandas as pd
+from mango_time_series.mango.utils.processing import aggregate_to_input
```

Comment:
Review if this script .py must be here or in the time-series-dashboard branch.
@antoniogonzalezsuarez